seqr/utils/communication_utils.py

```diff
@@ -41,3 +43,28 @@ def send_welcome_email(user, referrer):
         password_token=user.password,
     )
     user.email_user('Set up your seqr account', email_content, fail_silently=False)
+
+
+def send_load_data_email(user, guid, namespace, name, individual_ids):
+    for data_manager in User.objects.filter(is_staff = True):
+        email_content = """
+        Hi there {full_name}--
+    
+        Collaborator project manager {user_name}/email{email} has created a new project with a GUID:
+          {guid}
+        from an AnVIL workspace {namespace}/{name}.
+
+        The new project is ready for loading. The created individual IDs are in attached file.
+    
+        Thanks!
+        """.format(
+            full_name = data_manager.get_full_name(),
+            user_name = user.get_full_name(),
+            email = user.email,
+            guid = guid,
+            namespace = namespace,
+            name = name,
+        )
+        mail = EmailMessage('Requesting loading AnVIL workspace data', email_content, DEFAULT_FROM_EMAIL, [data_manager.email])
+        mail.attach('individual IDs', str(individual_ids), 'text/plain')
```

Comment:
I'm not sure about the format of the attached individual id file.

Reply:
make it a tsv with one column and each sample id is on its own row. The filename should be `{project.guid}_sample_ids.tsv`

Reply:
also the "attach" function should be able to guess the mimetype so I wouldn't specify it if I were you